app/models/article.js

```diff
@@ -0,0 +1,38 @@
+/**
```

Comment:
While setting up the test task using gulp, it was discovered some tests have already been written for models that were yet to be created, probably for a feature yet to be implemented, as a result the test task couldn't execute. The file was created for the expected model to execute the test pending a decision about the model.